tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -469,3 +484,34 @@ Delete Failed Build And Start New One
     ${build_name} =    Start New Build    namespace=${namespace}
     ...    buildconfig=${build_config_name}
     Wait Until Build Status Is    namespace=${namespace}    build_name=${build_name}
+
+Check Cluster Name Contain "Aisrhods" Or Not
+    [Documentation]     Return true if cluster name contains aisrhods and it not return false
+    ${cluster_id} =     Get Cluster ID
+    ${cluster_name} =    Get Cluster Name By Cluster ID  cluster_id=${cluster_id}
+    IF    "aisrhods" in "${cluster_name}"
+        ${return_value} =    Set Variable    ${TRUE}
+    ELSE
+        ${return_value} =    Set Variable    ${FAlSE}
+    END
```

Comment:
I think this could be something like
```
${return_value} =  Evaluate  "aisrhods" in "${cluster_name}"
[Return]  ${return_value}
```

You might also be able to return `Evaluate  "aisrhods" in "${cluster_name}"` but I'm not sure RF supports that

Reply:
worked
Thanks